src/poly/dcrt/sampler/trapdoor.rs

```diff
@@ -174,17 +175,18 @@ impl PolyTrapdoorSampler for DCRTPolyTrapdoorSampler {
             2_i64,
             false,
         );
+        debug_mem("After trap gen");
         let rlwe_trapdoor = dcrt_trapdoor.get_trapdoor_pair();
+        debug_mem("After get trapdoor pair");
         let nrow = size;
         let ncol = (&params.modulus_bits() + 2) * size;
         let public_matrix = DCRTPolyMatrix::from_poly_vec(
             params,
-            parallel_iter!(0..nrow)
-                .map(|i| {
-                    parallel_iter!(0..ncol).map(|j| dcrt_trapdoor.get_public_matrix(i, j)).collect()
-                })
+            (0..nrow)
+                .map(|i| (0..ncol).map(|j| dcrt_trapdoor.get_public_matrix(i, j)).collect())
                 .collect(),
         );
+        debug_mem("After get public matrix");
```

Comment:
i'd be curious to check time/memory difference with larger scale (e.g run middle param up to point reach trapdoor gen?) little worry if this non parallel loop might be too slow.

Reply:
> if this non parallel loop might be too slow.

I have the same concern.

Reply:
Ok, I can modify the PR and add the parallel feature then :)